src/hypertable.c

```diff
@@ -547,8 +547,8 @@ ts_hypertable_scan_with_memory_context(const char *schema, const char *table,
 									   bool tuplock, MemoryContext mctx)
 {
 	ScanKeyData scankey[2];
-	NameData schema_name = { 0 };
-	NameData table_name = { 0 };
+	NameData schema_name = { { 0 } };
+	NameData table_name = { { 0 } };
```

Comment:
This looks fine, but it makes more sense to just disable that warning. If the `NameData` structure changes (not likely, I know) this will start giving a warning again and I do not think the warning you mention is sensible to have at all.

Reply:
We inherit CFLAGS from postgres so we would have to filter those to not inherit this flag.